README.md

```diff
@@ -63,14 +63,14 @@ Questions and feedback are extremely welcome! Please understand that we won't be
 ## :dizzy: Features
 Our aim is to make the time series analysis ecosystem more interoperable and usable as a whole. sktime provides a __unified interface for distinct but related time series learning tasks__. It features [__dedicated time series algorithms__](https://www.sktime.org/en/stable/estimator_overview.html) and __tools for composite model building__ including pipelining, ensembling, tuning and reduction that enables users to apply an algorithm for one task to another.
 
-sktime also provides **interfaces to related libraries**, for example [scikit-learn], [statsmodels], [tsfresh], [PyOD] and [fbprophet], among others.
+sktime also provides **interfaces to related libraries**, for example [scikit-learn], [statsmodels], [tsfresh], [PyOD] and [fbprophet](https://facebook.github.io/prophet/), among others.
```

Comment:
Thanks! I think you don't need to change this line for it to work?

Reply:
Sure. I'll lool into it

Reply:
I have created another pull request where I have fixed this issue and also, in last section I have updated the formatting of the documentation.